src/app/modules/coding-test/components/quiz-test/quiz-test.component.ts

```diff
@@ -76,7 +76,6 @@ export class QuizTestComponent implements OnInit, OnDestroy {
       this.timerCount = `${prefix}${Math.floor(seconds / 60)}:${textSec}`;
 
       if (seconds == 0) {
-        console.log('finished');
```

Comment:
keep it as it is, Already covered under https://github.com/teamclairvoyant/clairthon-rising-hackatoons/pull/34